services/movie.py

```diff
@@ -0,0 +1,36 @@
+from django.db.models import QuerySet
+from typing import List, Optional
+
+from db.models import Movie
+
+
+def get_movies(genres_ids: Optional[List[int]] = None,
+               actors_ids: Optional[List[int]] = None) -> QuerySet:
+    movies_queryset = Movie.objects.all()
+
+    if genres_ids:
+        movies_queryset = movies_queryset.filter(genres__id__in=genres_ids)
+
+    if actors_ids:
+        movies_queryset = movies_queryset.filter(actors__id__in=actors_ids)
+
+    return movies_queryset.distinct()
+
+
+def get_movie_by_id(movie_id: int) -> Movie:
+    return Movie.objects.get(id=movie_id)
```

Comment:
it is a convention to use `pk` in ORM. By default primary key is `id` but there can be cases when it's not, so for this we use `pk` alias